frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelConversionWindowFilter.tsx

```diff
@@ -25,49 +25,44 @@ export function FunnelConversionWindowFilter(): JSX.Element {
+        <div style={{ marginTop: 16 }}>
+            <Row align="middle">
+                Conversion window
+                <Tooltip title="Only conversions within this timeframe of performing the initial step will be counted.">
+                    <InfoCircleOutlined className="info-indicator right" />
                 </Tooltip>
-            </span>
-            <Row className="funnel-options-conversion-window-inputs">
-                <InputNumber
-                    className="time-value-input"
-                    min={intervalBounds[0]}
-                    max={intervalBounds[1]}
-                    defaultValue={14}
-                    value={conversionWindow.funnel_window_interval}
-                    onChange={(funnel_window_interval) =>
-                        setConversionWindow({ funnel_window_interval: Number(funnel_window_interval) })
-                    }
-                    onBlur={loadResults}
-                    onPressEnter={loadResults}
-                />
-                <Select
-                    className="time-unit-input"
-                    defaultValue={FunnelConversionWindowTimeUnit.Day}
-                    dropdownMatchSelectWidth={false}
-                    value={conversionWindow.funnel_window_interval_unit}
-                    onChange={(funnel_window_interval_unit: FunnelConversionWindowTimeUnit) =>
-                        setConversionWindow({ funnel_window_interval_unit })
-                    }
-                    onBlur={loadResults}
-                >
-                    {options.map(({ value, label }) => (
-                        <Select.Option value={value} key={value}>
-                            {label}
-                        </Select.Option>
-                    ))}
-                </Select>
+            </Row>
+            <Row align="middle" gutter={8} style={{ marginTop: 8 }}>
+                <Col>
+                    <InputNumber
+                        min={intervalBounds[0]}
+                        max={intervalBounds[1]}
+                        defaultValue={14}
+                        value={conversionWindow.funnel_window_interval}
+                        onChange={(funnel_window_interval) => {
+                            setConversionWindow({ funnel_window_interval: Number(funnel_window_interval) })
+                            loadResults()
+                        }}
+                        onPressEnter={loadResults}
+                    />
+                </Col>
+                <Col>
+                    <Select
+                        defaultValue={FunnelConversionWindowTimeUnit.Day}
+                        dropdownMatchSelectWidth={false}
+                        value={conversionWindow.funnel_window_interval_unit}
+                        onChange={(funnel_window_interval_unit: FunnelConversionWindowTimeUnit) => {
+                            setConversionWindow({ funnel_window_interval_unit })
+                            loadResults()
```

Comment:
Thanks for taking a stab at this! I think the motive behind moving it from onChange into onBlur was that we didn't want to load results every time the value was changed because that would be expensive. Perhaps we can move this back into onBlur and wrap it in a if(prevValue !== nextValue) {} to trigger only when the value has changed

Reply:
I agree with @alexkim205, it also ends up feeling more intuitive when you do this `onBlur` (& on pressing enter key) than on every keystroke.